src/chains.ts

```diff
@@ -36,6 +36,18 @@ export const alephzeroTestnet: SubstrateChain = {
   faucetUrls: ['https://faucet.test.azero.dev'],
 }
 
+export const popNetwork: SubstrateChain = {
```

Comment:
Is "Pop Network" solely a testnet or will there be a mainnet endpoint at some point, too? If yes, I would suggest a naming convention like: `popMainnet` & `popTestnet`. Wdyt? cc @al3mart

Reply:
Not sure if there already existed a naming convention for this relay/parachain scheme, but it think it would be worth coming up with a common scheme, than we it would be clear what the Pop Network mainnet and testnet would be depending on the relay chain. 
Something like `Paseo/PopNetwork`, `Polkadot/PopNetwork`.

Thinking this even further it would be actually pretty cool imho to use some sort of defined standard for a simple string representation of full XCM multilocation for the chain. Than it would be more like a proper URI which can be used to create xcm destinations, cc @daanvdplas.

https://paritytech.github.io/xcm-docs/fundamentals/multilocation/index.html

Reply:
That would be awesome @peetzweg.

@wottpal this is pointing to the testnet, but it is meant to exist on Pokadot :) So making sure users can differentiate between both will be handy.

`popNetwork` and `popTestnet` might work perfectly fine! Though, if a common scheme is to be used we are more than happy to follow that. Personally, I like the idea of prepending the relay name to the parachain name, it reads very clear.

Reply:
I changed it to `popNetworkTestnet` and `pop-network-testnet` for now. We might come up with a more general naming scheme for everything in the future but should for sure not be part of this pr. :)

https://github.com/scio-labs/use-inkathon/blob/bc60f403d63b5089bc4052d8e85eee040360db18/src/chains.ts#L39-L40